contiamo/data.py

```diff
@@ -34,22 +46,35 @@ def _make_request(self, url, file_object=None):
 
     return json_response
 
-  def post_df(self, url, dataframe):
+  def post_df(self, url, dataframe, include_index=False):
     with tempfile.NamedTemporaryFile() as f:
+
+      if include_index:
+        dataframe.reset_index(inplace=True, drop=False)
+
```

Comment:
I just realized that we have a problem: if we modify the dataframe in place, then the dataframe that was passed by the user as argument will be modified (in the user's environment). In other words, the function has a side-effect. This should fix it: `dataframe = dataframe.reset_index(drop=False)`.

Also, can you move the `if` block outside of the `with` block?